### ISSUE TITLE
Remove no trace mentions from all <a> links

### ISSUE THREAD

**Issue by ConsenSys-Academy** [ConsenSys-Academy/basic-training#16]
[Emmet expansions](https://docs.emmet.io/) for links with `target="_blank"` also add `rel="noopener noreferrer"` to all href tags, like this example here:

```html
<a href="https://github.com/ohmyzsh/ohmyzsh/wiki/Themes" target="_blank" rel="noopener noreferrer">this page</a>
```

It'd be great to do a "Find All" and delete in VSCode or similar of all the `rel="noopener noreferrer"` that are in the course, as they mask where the user is coming from. 

Someone will have to clone the repo locally on their machine and run Find and Replace on local editor

**Comment by jackm9367** [comment]
Hey @ConsenSys-Academy - Would love to work on this one. 

Feel free to assign it to me, I'll get started now. Cheers!

**Comment by jackm9367** [comment]
Sorry @Her @ConsenSys-Academy - I accidentally stole the issue (there was already a PR here: #19)

**Comment by jackm9367** [comment]
@ConsenSys-Academy Should this issue be closed after merging #19?
